pkg/mounter/mounter-s3fs_test.go

```diff
@@ -5,28 +5,30 @@ import (
 	"os"
 	"testing"
 
+	"github.com/IBM/ibm-object-csi-driver/pkg/constants"
 	mounterUtils "github.com/IBM/ibm-object-csi-driver/pkg/mounter/utils"
 	"github.com/stretchr/testify/assert"
 )
 
 var (
 	secretMap = map[string]string{
-		"cosEndpoint":        "test-endpoint",
-		"locationConstraint": "test-loc-constraint",
-		"bucketName":         "test-bucket-name",
-		"objPath":            "test-obj-path",
-		"accessKey":          "test-access-key",
-		"secretKey":          "test-secret-key",
-		"apiKey":             "test-api-key",
-		"kpRootKeyCRN":       "test-kp-root-key-crn",
-		"uid":                "test-uid",
+		"cosEndpoint":             "test-endpoint",
+		"locationConstraint":      "test-loc-constraint",
+		"bucketName":              "test-bucket-name",
+		"objPath":                 "test-obj-path",
+		"accessKey":               "test-access-key",
+		"secretKey":               "test-secret-key",
+		"apiKey":                  "test-api-key",
+		"kpRootKeyCRN":            "test-kp-root-key-crn",
+		"uid":                     "test-uid",
+		constants.CipherSuitesKey: "default",
```

Comment:
In our code, we are not adding cipher_suite in secretMap. I think we also should not add this to secretMap in UT. Please check whats correct place for cipher_suite as per testcase.